proto/server/pi_server.cpp

```diff
@@ -538,28 +538,37 @@ pi::server::ServerData *server_data;
 
 extern "C" {
 
-void PIGrpcServerRunAddr(const char *server_address) {
+void PIGrpcServerRunAddrGnmi(const char *server_address, void* gnmi_service) {
   server_data = new ::pi::server::ServerData();
   server_data->server_address = std::string(server_address);
   auto &builder = server_data->builder;
   builder.AddListeningPort(
     server_data->server_address, grpc::InsecureServerCredentials(),
     &server_data->server_port);
   builder.RegisterService(&server_data->pi_service);
+  if (gnmi_service != nullptr) {
+    server_data->gnmi_service = std::unique_ptr<gnmi::gNMI::Service>(
+            static_cast<gnmi::gNMI::Service*>(gnmi_service));
```

Comment:
style nit: s/`static_cast<gnmi::gNMI::Service*>`/`static_cast<gnmi::gNMI::Service *>`

Reply:
ok, fixed the style. thanks for reviewing